src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task{
+
+    private String deadline;
+
+    public Deadline(String name, String deadline) {
+        super(name);
+        this.deadline= deadline;
```

Comment:
```suggestion
        this.deadline = deadline;
```
Maybe add a space to improve readability. I noticed this in other parts of the code as well